analysis/wwz/wwz4l.py

```diff
@@ -140,14 +140,54 @@ def __init__(self, samples, wc_names_lst=[], hist_lst=None, ecut_threshold=None,
 
         }
 
+        # Add histograms to dictionary that will be passed on to dict_accumulator
+        dout = {}
+        for dense_axis_name in self._dense_axes_dict.keys():
+            dout[dense_axis_name] = hist.Hist(
+                hist.axis.StrCategory([], growth=True, name="process", label="process"),
+                hist.axis.StrCategory([], growth=True, name="category", label="category"),
+                hist.axis.StrCategory([], growth=True, name="systematic", label="systematic"),
+                self._dense_axes_dict[dense_axis_name],
+                storage="weight", # Keeps track of sumw2
+                name="Counts",
+            )
+
+        if siphon_bdt_data:
+            # Adding list accumulators for BDT output variables and weights
+            list_output_names = [
+                "bdt_of_wwz_list",
+                "bdt_of_zh_list",
+                "bdt_of_bkg_list",
+                "bdt_of_proc_list",
+                "bdt_of_wgt_list",
+                "bdt_of_evt_list",
+                "bdt_sf_wwz_list",
+                "bdt_sf_zh_list",
+                "bdt_sf_bkg_list",
+                "bdt_sf_proc_list",
+                "bdt_sf_wgt_list",
+                "bdt_sf_evt_list",
+            ]
+            for list_output_name in list_output_names:
+                dout[list_output_name] = processor.list_accumulator([])
+
+            # Also adding BDT input variable list accumulators
+            for of_bdt_var_name in get_ec_param("of_bdt_var_lst"):
```

Comment:
The input lists are nearly identical. Would it make sense to just make a `set` of the two lists? Or is there a reason why you want to specifically have the OF and SF ones kept track of separately?

Reply:
Because the set of events that fall into each branch are different

Reply:
Ahh, ok, got it. I was not looking at how they get used.